web_src/less/_base.less

```diff
@@ -158,6 +158,7 @@
   --color-tooltip-bg: #000000f0;
   --color-tooltip-text: #ffffff;
   --color-header-bar: #ffffff;
+  --color-label-active: #d0d0d0;
```

Comment:
As it is only used for the background, should we perhaps call it `color-background-label-active` or `color-background-active-label`?

Reply:
Hmm yes, we have not been really consistent with labeling background colors in the past, but I guess we could start doing that now.

Reply:
Renamed.